javascript/futurism_channel.js

```diff
@@ -33,6 +33,10 @@ export const createSubscription = consumer => {
         CableReady.perform(data.operations, {
           emitMissingElementWarnings: false
         })
+        
+        document.dispatchEvent(
+          new CustomEvent('futurism:appear', { bubbles: true, cancelable: true })
```

Comment:
Awesome, thanks a ton!

Please just rewrite to `futurism:appeared`, as `appear` is already taken.

Reply:
For the prettier-standard error, please add `CustomEvent` to globals like here: https://github.com/julianrubisch/futurism/blob/8cf79e4a966bd4da0b1929d433c08b5cb3bc7b0b/javascript/elements/futurism_utils.js#L1